integration/helpers/utils.ts

```diff
@@ -69,71 +37,69 @@ export function describeIf(condition: boolean) {
   return condition ? describe : describe.skip;
 }
 
-export const keys = {
-  backspace: "\uE003"
-};
+async function launchElectron() {
+  let error;
 
-export async function setup(): Promise<Application> {
-  const app =  new Application({
-    path: AppPaths[process.platform], // path to electron app
-    args: [],
-    startTimeout: 60000,
-    waitTimeout: 10000,
-    env: {
-      CICD: "true"
+  for (let i = 0; i < 10; i += 1) {
+    try {
+      return await electron.launch({
```

Comment:
This is missing temporary path for user data. Now it's pretty dangerous (and probably flaky) to run integration tests because it will read/modify users real environment. Most likely `--integration-testing` flag can be removed after we fix this.

Reply:
Not quite, but we don't need to pass it to the renderer if we change the user data path. I initially introduced this flag so as to not block the app from quitting (which we do under normal circumstances).

I guess a proper solution would be to send a SIGTERM to the process, I might look into that.